stat.c

```diff
@@ -3070,13 +3071,16 @@ static int add_iops_samples(struct thread_data *td, struct timespec *t)
 int calc_log_samples(void)
 {
 	struct thread_data *td;
-	unsigned int next = ~0U, tmp;
+	unsigned int next = ~0U, tmp = 0, next_mod = 0, log_avg_msec_min = 0xFFFFFFFF;
```

Comment:
Doing a quick search of the code base for the following:

```
% git grep '== ~0' | wc -l 
       1
% git grep '== -1U' | wc -l
      27
% git grep -i '== 0xFF' | wc -l 
       0
% git grep '== .*INT_MAX' | wc -l   
       5
```

It looks like there's a preference for using `-1U` as a magic maximum sentinel. Could you swap this to use that?

Reply:
Changed 0xffffffff to -1U.